app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -42,6 +43,11 @@ protected void onCreate(Bundle savedInstanceState) {
             setupDrawerContent(navigationView);
         }
 
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            getWindow().setStatusBarColor(getResources().getColor(R.color.color_primary));
```

Comment:
Yes it totally is right. I just used the color from where Manan has placed
it right now and using everywhere else. I am assuming he has a theming
scheme in mind.
On Jun 30, 2015 10:09 PM, "Mohit Kanwal" <notifications@github.com> wrote:

> In app/src/main/java/org/fossasia/openevent/activities/MainActivity.java
> <https://github.com/fossasia/open-event-android/pull/47#discussion_r33594125>
> :
>
> > @@ -42,6 +43,11 @@ protected void onCreate(Bundle savedInstanceState) {
> >              setupDrawerContent(navigationView);
> >          }
> >
> > +        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
> > +            getWindow().setStatusBarColor(getResources().getColor(R.color.color_primary));
>
> I believe setting this in the theme palette as colorPrimary etc. Results
> in less code and more separation.
>
> <style name="Theme.MyTheme" parent="Theme.AppCompat.Light">
>     <!-- colorPrimary is used for the default action bar background -->
>     <item name="colorPrimary">@color/my_awesome_color</item>
>
>     <!-- colorPrimaryDark is used for the status bar -->
>     <item name="colorPrimaryDark">@color/my_awesome_darker_color</item>
>
>     <!-- colorAccent is used as the default value for colorControlActivated         which is used to tint widgets -->
>     <item name="colorAccent">@color/accent</item>
>
>     <!-- You can also set colorControlNormal, colorControlActivated         colorControlHighlight & colorSwitchThumbNormal. -->
> </style>
>
> Please refer to https://chris.banes.me/2014/10/17/appcompat-v21/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-android/pull/47/files#r33594125>.
>